.travis.yml

```diff
@@ -28,6 +28,9 @@ matrix:
     env: SKIP_RELEASE=true
   - jdk: openjdk14
     env: SKIP_RELEASE=true MOCK_MAKER=mock-maker-inline
+  - jdk: openjdk11
+    script: ./gradlew spotlessCheck
+    after_success: true
```

Comment:
What does ^^ do?

Reply:
To make sure we don't run the coverage report task, since that's not what spotless generates. I will add a clarifying comment.